Signal/src/view controllers/ContactsPicker.swift

```diff
@@ -78,28 +80,15 @@ open class ContactsPicker: UITableViewController, UISearchResultsUpdating, UISea
         registerContactCell()
         initializeBarButtons()
         reloadContacts()
-        initializeSearchBar()
+        updateSearchResults(searchText: "")
+        self.tableView.reloadData()
```

Comment:
This `reloadData` is redundant because we're doing it at the end of updateSearchResults.